release-please-config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "bootstrap-sha": "b82a08523d0bb5d9e27f87f10924296ca745f236",
+    "exclude-paths": [
+        ".git",
+        ".devcontainer",
+        ".github",
+        ".gradle",
+        ".vscode",
+        "scripts",
+        "gradle"
+    ],
+    "release-type": "simple",
+    "include-component-in-tag": false,
+    "include-v-in-tag": true,
+    "packages": {
+        ".": {
```

Comment:
I think you should make use of multiple packages for each library in case one of the libraries needs to release a patch version

Reply:
the releasing is setup in such a way today that we cannot release any package independently. We'd need to update the CD to support such a scenario.

Reply:
We intend to bump all the packages when one changes to keep a consistent version across the board. Reduces confusion about what version of a kiota package is compatible with what etc.

Reply:
@baywet thoughts about creating multiple GitHub releases for each package?

Would mean we include the component name in the tags e.g. `abstractions-v1.6.0` to prevent conflicts when we update each package's GitHub release with its relevant JAR.

Reply:
(Ah, forgot to refresh :). Deleting duplicate comments)

Reply:
it's not just about the GH release, it's also about the upload to maven central. We'd need to somehow detect which packages have been bumped, and only perform the upload for those. I'm not saying it's impossible, just more work.
And we've already gotten the feedback that:
1. misaligned versions are confusions
2. updating all dependencies is not the initial burden we thought, especially when versions are aligned

And we've invested in the bundle package.

This is why I'm suggesting we setup RP in this PR (initial scope) and avoid expanding the scope to change a bunch of things in the versioning, CD, etc... At least until we get further feedback that we're producing too many releases.